simulation/utils/simulation_pymunk_utils.py

```diff
@@ -34,8 +37,8 @@ def create_agent(position_x, color, max_distance_from_threshold=100):
     min_pos = position_x - max_distance_from_threshold
     max_pos = position_x + max_distance_from_threshold
 
-    radius = 12
-    mass = 10
+    radius = AGENTS_RADIUS
+    mass = AGENTS_MASS
```

Comment:
Zmieniłbym wszędzie `agents` na `bots` czy coś podobnego. Może się mylić z agentem. Tym co operuje na środowisku. Ale to można zrobić osobnego pull requesta na to.

Reply:
👍